### ISSUE TITLE
🐛 BUG: Local server not starting

### ISSUE THREAD

**Issue by cloudelice** [cloudflare/wrangler2#2334]
### What version of `Wrangler` are you using?

2.6.0

### What operating system are you using?

Mac

### Describe the Bug

I just upgraded from 2.5.0 to 2.6.0 and it seems that local dev port is not working anymore when running `yarn wrangler dev --local`.  
Maybe because there was a debugger issue in 2.5.0 and lower versions (address already in use) that seems to be fixed in 2.6.0. This issue being fixed, when I run `yarn wrangler dev --local` it blocks here without starting the web server:

![](https://media.discordapp.net/attachments/779390076219686943/1049449354714427493/image.png)

**Comment by threepointone** [comment]
can confirm, --local mode doesn't start anymore

**Comment by tulup-conner** [comment]
Same here with `wrangler pages dev`

**Comment by KianNH** [comment]
The first issue when you open DevTools is this:

![image](https://user-images.githubusercontent.com/94662631/205858240-928a442a-1c55-421c-ba56-e2f833ba0272.png)

If you do `npm i @miniflare/tre` then it becomes this:

![image](https://user-images.githubusercontent.com/94662631/205858294-7a7574fc-976b-4ae3-bfb2-7462bef4b05b.png)

Doing `npm i @miniflare/tre@3.0.0-next.8` appears to fix the issue.

![image](https://user-images.githubusercontent.com/94662631/205858426-0f516da9-2c94-4948-8b3c-e3b1ac9eac6c.png)

**Comment by KianNH** [comment]
```sh
➜  test-worker rm -rf node_modules

➜  test-worker rm -rf package-lock.json

➜  test-worker rm -rf package.json

➜  test-worker npm i wrangler
npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead

added 85 packages, and audited 220 packages in 15s

43 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

➜  test-worker npx wrangler dev --local
 ⛅️ wrangler 2.6.0
-------------------
Want to try out the next version of local mode using the open-source Workers runtime?
Switch out --local for --experimental-local and let us know what you think at https://discord.gg/cloudflaredev !
Using vars defined in .dev.vars
Your worker has access to the following bindings: ...
⎔ Starting a local server...
╭───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [b] open a browser, [d] open Devtools, [l] turn off local mode, [c] clear console, [x] to exit                                            │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Debugger listening on ws://127.0.0.1:60825/74021e4f-1bba-4673-87e3-d2eff97c123c
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Waiting for the debugger to disconnect...
⎔ Shutting down local server.

➜  test-worker npm i @miniflare/tre@3.0.0-next.8

added 54 packages, and audited 274 packages in 36s

55 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

➜  test-worker npx wrangler dev --local
 ⛅️ wrangler 2.6.0
-------------------
Want to try out the next version of local mode using the open-source Workers runtime?
Switch out --local for --experimental-local and let us know what you think at https://discord.gg/cloudflaredev !
Using vars defined in .dev.vars
Your worker has access to the following bindings: ...
⎔ Starting a local server...
╭───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [b] open a browser, [d] open Devtools, [l] turn off local mode, [c] clear console, [x] to exit                                            │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Debugger listening on ws://127.0.0.1:60842/dde6f76a-b2a5-4a9a-b314-0b429770e194
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
[mf:inf] Worker reloaded! (1.20KiB)
[mf:inf] Listening on 0.0.0.0:8787
[mf:inf] - http://127.0.0.1:8787
[mf:inf] - http://192.168.1.87:8787
[mf:inf] - http://192.168.192.165:8787
[mf:inf] Updated `Request.cf` object cache!
```

**Comment by penalosa** [comment]
Could you give this version of wrangler a go to see if it fixes the issue?

```
npm install --save-dev https://prerelease-registry.devprod.cloudflare.dev/prs/2338/npm-package-wrangler-2338
```

**Comment by KianNH** [comment]
Same issue here.

```sh
➜  test-worker npx wrangler dev --local
 ⛅️ wrangler 2.6.0
-------------------
Want to try out the next version of local mode using the open-source Workers runtime?
Switch out --local for --experimental-local and let us know what you think at https://discord.gg/cloudflaredev !
Using vars defined in .dev.vars
Your worker has access to the following bindings: ...
⎔ Starting a local server...
╭───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [b] open a browser, [d] open Devtools, [l] turn off local mode, [c] clear console, [x] to exit                                            │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Debugger listening on ws://127.0.0.1:65026/85ff97b1-864f-409c-92e1-090c9dd6c6fc
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Waiting for the debugger to disconnect...
⎔ Shutting down local server.

➜  test-worker npm install --save-dev https://prerelease-registry.devprod.cloudflare.dev/prs/2338/npm-package-wrangler-2338
npm WARN idealTree Removing dependencies.wrangler in favor of devDependencies.wrangler

changed 1 package, and audited 103 packages in 928ms

11 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

➜  test-worker npx wrangler dev --local
 ⛅️ wrangler 0.0.0-27c16677 (update available 0.0.0-271aece0)
-----------------------------------------------------------------------
Want to try out the next version of local mode using the open-source Workers runtime?
Switch out --local for --experimental-local and let us know what you think at https://discord.gg/cloudflaredev !
Using vars defined in .dev.vars
Your worker has access to the following bindings: ...
⎔ Starting a local server...
╭───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [b] open a browser, [d] open Devtools, [l] turn off local mode, [c] clear console, [x] to exit                                            │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Debugger listening on ws://127.0.0.1:65068/12b065d3-4271-4a40-86c0-686a1fbedd3b
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Waiting for the debugger to disconnect...
⎔ Shutting down local server.

➜  test-worker npm i --save-dev @miniflare/tre@3.0.0-next.8

added 60 packages, and audited 163 packages in 40s

23 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

➜  test-worker npx wrangler dev --local
 ⛅️ wrangler 0.0.0-27c16677 (update available 0.0.0-271aece0)
-----------------------------------------------------------------------
Want to try out the next version of local mode using the open-source Workers runtime?
Switch out --local for --experimental-local and let us know what you think at https://discord.gg/cloudflaredev !
Using vars defined in .dev.vars
Your worker has access to the following bindings: ...
⎔ Starting a local server...
╭───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [b] open a browser, [d] open Devtools, [l] turn off local mode, [c] clear console, [x] to exit                                            │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Debugger listening on ws://127.0.0.1:65163/e2144634-b733-4c37-9231-05bdb7c77d8b
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
[mf:inf] Worker reloaded! (1.20KiB)
[mf:inf] Listening on 0.0.0.0:8787
[mf:inf] - http://127.0.0.1:8787
[mf:inf] - http://192.168.1.87:8787
[mf:inf] - http://192.168.192.165:8787
[mf:inf] Updated `Request.cf` object cache!
```

**Comment by rozenmd** [comment]
There's a fix for this in `wrangler@2.6.1`
